app/components/App.jsx

```diff
@@ -25,6 +26,7 @@ const App = () => (
     <Layout>
       <Switch>
         <Route exact path="/" component={anonymousRoute(Home)} />
+        <Route exact path="/conduct" component={anonymousRoute(Conduct, false)} />
```

Comment:
If you don't want the redirect behaviour, just remove the `anonymousRoute` HOC. It's the only thing it does.

`component={Conduct}` should work, or am I missing something?

Reply:
I think you're right and I might have violently derped here. Will check later.